src/main/java/elements/LinkImpl.java

```diff
@@ -0,0 +1,27 @@
+package elements;
+
+
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+import util.WaitUtils;
+
```

Comment:
Please make sure number of space lines between imports and class/java docs declaration is the same in every class

Reply:
checked